packages/core/src/external/carequality/ihe-gateway-v2/saml/saml-client.ts

```diff
@@ -44,14 +45,17 @@ export async function sendSignedXml({
   trustedKeyStore: string;
 }): Promise<string> {
   const agent = new https.Agent({
-    rejectUnauthorized: true,
+    rejectUnauthorized: false,
     cert: samlCertsAndKeys.certChain,
     key: samlCertsAndKeys.privateKey,
     passphrase: samlCertsAndKeys.privateKeyPassword,
     ca: trustedKeyStore,
+    ciphers: "DEFAULT:!DH",
+    secureOptions: constants.SSL_OP_ALLOW_UNSAFE_LEGACY_RENEGOTIATION,
```

Comment:
these two changes allow us to work with several endpoints with more legacy encryption standards